ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -409,6 +410,39 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
                 dispatchSessionExpired,
                 true
               )}
+            {isDevEnv &&
+              this.debugListItem(
+                I18n.t("profile.main.clearAsyncStorage"),
+                () => {
+                  void AsyncStorage.clear();
+                },
+                true
+              )}
+            {isDevEnv &&
+              this.debugListItem(
+                I18n.t("profile.main.dumpAsyncStorage"),
+                () => {
+                  /* eslint-disable no-console */
+                  console.log("[DUMP START]");
+                  AsyncStorage.getAllKeys()
+                    .then(keys => {
+                      console.log(`\tAvailable keys: ${keys.join(", ")}`);
+                      return Promise.all(
+                        keys.map(key =>
+                          AsyncStorage.getItem(key).then(value => {
+                            // if (/(features)|(euCovidCert)|(mvl)/.test(key)) {
```

Comment:
could we remove this commented code?

Reply:
Good catch! Fixed [here](https://github.com/pagopa/io-app/pull/3681/commits/186aeb787ba33b385cc380ea2d76af770bf617fd)